api/users.js

```diff
@@ -19,16 +19,30 @@ usersRouter.use((req, res, next) => {
 });
 
 usersRouter.get("/", async (req, res, next) => {
```

Comment:
since users have "sensitive" data like email, it would be suggested to have requireAdmin